index.js

```diff
@@ -2,21 +2,337 @@ const openBtn = document.querySelector('#open-menu-btn');
 const closeBtn = document.querySelector('#close-menu-btn');
 const menu = document.querySelector('.mobile-menu');
 const listBtns = document.querySelectorAll('.listBtns');
+const body = document.querySelector('body');
 
-function createClass (elem, className) {
-    elem.classList.add(className)
+function creatPopUp(
+  name,
+  description,
+  img,
+  techs,
+  live = 'SeeLive',
+  source = 'SeeSource',
+) {
+  const wrapper = document.createElement('div');
+  const contentContainer = document.createElement('div');
+  const headerContainer = document.createElement('div');
+  const header = document.createElement('h2');
+  header.textContent = name;
+  const closeBtn = document.createElement('a');
+  closeBtn.textContent = 'x';
+  const image = document.createElement('img');
+  image.src = img;
+  const paragraph = document.createElement('p');
+  paragraph.textContent = description;
+  const tags = document.createElement('div');
+  const anchor1 = document.createElement('a');
+  const anchor2 = document.createElement('a');
+  const anchor3 = document.createElement('a');
+  const anchor4 = document.createElement('a');
+  // anchor1.textContent = techs[0];
+  // anchor2.textContent = techs[1];
+  // anchor3.textContent = techs[2];
+  // anchor4.textContent = techs[3];
+  [anchor1.textContent, anchor2.textContent, anchor3.textContent, anchor4.textContent] = techs;
+  anchor1.href = '#';
+  anchor2.href = '#';
+  anchor3.href = '#';
+  anchor4.href = '#';
+  const btns = document.createElement('div');
+  const btnLive = document.createElement('button');
+  const btnSrc = document.createElement('button');
+  btnLive.textContent = live;
+  btnSrc.textContent = source;
```

Comment:
- [ ] As requested by the former code reviewer, which is also part of the project requirements, which is an important skill for a font-end developer, the alignment of this buttons in the desktop version(see image below) looks different from that in the original design, please kindly fix that. 

***The design***
![theD](https://user-images.githubusercontent.com/65852621/136300358-a7908113-1ca8-43d6-b653-9926a0ce911d.png)

***Your implementation***
![theI](https://user-images.githubusercontent.com/65852621/136300372-0c2b8975-38ab-4f42-9e6a-1a9f7dff1aa3.png)